processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -553,6 +554,20 @@ private long getMaxTimeMillis()
     }
   }
 
+  public List<DimDim> getDimensionValues()
```

Comment:
`getDimensionIndex(String dimension)` is also unused, i know this PR did not introduce that but can you pls remove that as well.